### PR TITLE
 React People Table (Advanced)

### PR DESCRIPTION
**Pull request by svyatoslavPy**
[DEMO LINK](https://svyatoslavPy.github.io/react_people-table-advanced/)

### CODE REVIEW

File: src/utils/getSortedPeople.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  if (sort === Sorts.name || sort === Sorts.sex) {
+    currentPeople = currentPeople.sort((a, b) =>
+      a[sort].localeCompare(b[sort]),
+    );
+  }
+
+  if (sort === Sorts.born || sort === Sorts.died) {
+    currentPeople = currentPeople.sort((a, b) => a[sort] - b[sort]);
+  }
```
Comment (etojeDenys) [inline src/utils/getSortedPeople.ts:22]: you can simplify it using the switch/case

File: src/utils/getFilteredPeople.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// import { useSearchParams } from 'react-router-dom';
```
Comment (etojeDenys) [inline src/utils/getFilteredPeople.ts:1]: looks redundant

**Review by etojeDenys — CHANGES_REQUESTED** [review]
do not reset query params on person click
https://github.com/mate-academy/react_people-table-advanced/assets/70220667/fd01abae-1a36-47d0-a8f0-6907b135da29


2. looks different from the working example
image

**Review by l4st1m0za — APPROVED** [review]
Good job!

**Review by lerastarynets — APPROVED** [review]
lgtm 🚀

**Pull request closed without merging**